File: rust/tests/exists_test.rs
// Generated existence test: the SDK constructs in test mode.

use bluefin_decryptx_p2pe_sdk::{test_sdk, Value};

#[test]
fn exists_test_mode() {
    let testsdk = test_sdk(Value::Noval, Value::Noval);
    assert_eq!(*testsdk.mode.borrow(), "test");
}

File: swift/Tests/BluefinDecryptxP2peSdkTests/ExistsTest.swift
// BluefinDecryptxP2pe SDK exists test.

import XCTest

@testable import BluefinDecryptxP2peSdk

final class ExistsTest: XCTestCase {
  func testMode() {
    let testsdk = BluefinDecryptxP2peSDK.testSDK(nil, nil)
    XCTAssertEqual(testsdk.mode, "test")
  }
}
